README.md

```diff
@@ -509,6 +515,7 @@ Services to securely store your Docker images.
 * [Docker on Windows behind a firewall](https://toedter.com/2015/05/11/docker-on-windows-behind-a-firewall/) by [@kaitoedter](https://twitter.com/kaitoedter)
 * [Docker Tips](http://www.mervine.net/notes/docker-tips) by [@jmervine](https://github.com/jmervine)
 * [Dockerfile Generator](http://jrruethe.github.io/blog/2015/09/20/dockerfile-generator/) (ruby script)
+* [Everyday Docker Hacks](https://codefresh.io/blog/everyday-hacks-docker/) by [@alexei-led](github alexei ledenev)
```

Comment:
Nice, but some of these hacks are already available as cheat sheets. Under this category, we already have two.

In addition, this [`Docker Cheat Sheet`](https://github.com/wsargent/docker-cheat-sheet#tips) has even referenced to your link.